src/main/java/core/basesyntax/stategy/handler/ReturnOperation.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.stategy.handler;
+
+import core.basesyntax.db.Storage;
```

Comment:
The package declaration should not be separated from the import statements by an empty line.